.github/workflows/ci.yml

```diff
@@ -0,0 +1,267 @@
+        with:
+          path: |
+            .plz-cache
+          key: ${{ runner.os }}-plz-v2-${{ hashFiles('**/BUILD', '**/BUILD.plz', '**/.plzconfig*') }}
+          restore-keys: |
+            ${{ runner.os }}-plz-v2-
+
+      - name: Go module cache
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-mod-
+
+      - name: Go build cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/go-build
+          key: ${{ runner.os }}-go-build-test-${{ hashFiles('**/*.go', '**/go.mod', '**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-build-test-
+
+      - name: Test
+        run: |
+          ./pleasew test //...
+          ./pleasew run ///pleasings2//tools/go:gotestsum -- -- ./...
+
+      - name: Save Please log output
+        uses: actions/upload-artifact@v1
+        with:
+          name: test-log
+          path: plz-out/log
+
+  integration-test:
+    name: Integration Test
+    runs-on: ubuntu-latest
+
+    needs: build
+
+    services:
+      vault:
+        image: vault:1.4.2
+        env:
+          SKIP_SETCAP: true
+          VAULT_DEV_ROOT_TOKEN_ID: 227e1cce-6bf7-30bb-2d2a-acc854318caf
+        ports:
+          - 8200
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Please cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            .plz-cache
+          key: ${{ runner.os }}-plz-v2-${{ hashFiles('**/BUILD', '**/BUILD.plz', '**/.plzconfig*') }}
+          restore-keys: |
+            ${{ runner.os }}-plz-v2-
+
+      - name: Go module cache
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-mod-
+
+      - name: Go build cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/go-build
+          key: ${{ runner.os }}-go-build-integration-test-${{ hashFiles('**/*.go', '**/go.mod', '**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-build-integration-test-
+
+      - name: Start kind cluster
+        run: |
+          ./pleasew run ///pleasings2//tools/k8s:kind -- create cluster --kubeconfig $HOME/.kube/config
+          echo "::set-env name=KUBECONFIG::$HOME/.kube/config"
+
+      - name: Setup Helm
+        run: |
+          ./pleasew run ///pleasings2//tools/k8s:kubectl -- create serviceaccount tiller --namespace kube-system
+          ./pleasew run ///pleasings2//tools/k8s:kubectl -- create clusterrolebinding --clusterrole cluster-admin --serviceaccount=kube-system:tiller tiller
+          ./pleasew run :helm2 -- init --service-account tiller
```

Comment:
**Question**: why are we using Helm2 here?

Reply:
Because we still have helm 2 support that requires tiller on the cluster.